### ISSUE TITLE
Could you clarify `with proper learning rate scaling across batch size` in "Intriguing Properties of Contrastive Losses"?

### ISSUE THREAD

**Issue by nzw0301** [google-research/simclr#166]
Dear the authors of the paper entitled "Intriguing Properties of Contrastive Losses",

First of all, thank you for sharing the experimental codes and conducting comprehensive experiments in the SimCLR paper!

As in [the first item of `README`'s known issues](https://github.com/google-research/simclr#known-issues), "Intriguing Properties of Contrastive Losses" paper shows interesting experimental results, where smaller mini-batch size works well in practice on Table 2. However, I could not find a detailed hyper-parameter configuration to get these results.

Would you mind clarifying the configuration such as the concrete hyper-parameter or hyper-parameter space?

Best regards,

**Comment by chentingpc** [comment]
Here's a set of key hparam we used:
`learning_rate: 0.075, learning_rate_scaling: sqrt, proj_out_dim: 128, temperature: 0.2`
hope this helps.

**Comment by nzw0301** [comment]
Many thanks for sharing the parameters! Can I ask the CIFAR-10's hyper-parameters as well?

**Comment by chentingpc** [comment]
For CIFAR that would be
`learning_rate: 0.2, learning_rate_scaling: sqrt, num_proj_layers: 2, proj_out_dim: 64, temperature: 0.2`

**Comment by nzw0301** [comment]
Again, thank you so much for sharing it!
